frontend/src/components/profile/components/user-info/user-info.tsx

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { useForm } from 'react-hook-form';
 import clsx from 'clsx';
 import { IUser } from 'common/interfaces/user';
-import { BindingCb, InputChangeEvent } from 'common/types';
+import { BindingCb, InputChangeEvent, RootState } from 'common/types';
 import { getFormattedDate } from 'helpers';
-import { IUserTypeDoctor } from 'common/interfaces';
+import { IOption, IProfession, IUserTypeDoctor } from 'common/interfaces';
 import { ProfileActionCreator } from 'store/slices';
-import { Button } from 'components/common';
+import { Button, Select } from 'components/common';
 import Documents from '../documents/documents';
 import {
   ButtonColor,
   ButtonStyleType,
   ButtonIcon,
   DateFormat,
+  InputColor,
+  ButtonType,
+  ProfessionKey,
 } from 'common/enums';
 import styles from './styles.module.scss';
 import defaultAvatar from 'assets/images/default-avatar.svg';
 import AddClinic from '../add-clinic/add-clinic';
 
 const DEFAULT_FILE_IDX = 0;
+const DEFAULT_PROFESSION_VALUE = {
```

Comment:
Add type and move it to the `frontend/src/components/profile/components/user-info/common/constants`